services/mock-blockchain/src/mockNFTContract.ts

```
@@ -0,0 +1,127 @@
+import { NFTContractInterface, NFTToken } from './interfaces';
+import { simulateDelay, simulateError } from './utils';
+
+export class MockNFTContract implements NFTContractInterface {
+  private tokens: Map<string, NFTToken> = new Map();
+  private ownerTokens: Map<string, Set<string>> = new Map();
+  private nextTokenId: number = 1;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)